js/views/pageNavVw.js

```diff
@@ -139,6 +142,13 @@ module.exports = Backbone.View.extend({
       if(self.model.get('beenSet')){
         self.$el.find('.js-homeModal').hide();
       }
+      self.notificationsPanel = new notificationsPanelView({
+        parentEl: '#notificationsPanel',
+        socketView: self.socketView,
+        serverUrl: self.options.model.attributes.serverUrl
```

Comment:
To be consistent this should be self.options.model.get('serverUrl'), but this works too.